bin.js

```diff
@@ -64,8 +64,14 @@ if (opts['upload-all']) {
 }
 
 function uploadFiles (files) {
-  buildLog('Uploading ' + files.length + ' prebuilds(s) to Github releases')
-  upload(extend(opts, {files: files}), function (err, result) {
+    // NOTE(robinwassen): Only include unique files
+    // See: https://github.com/prebuild/prebuild/issues/221
+    var uniqueFiles = files.filter(function (file, index) {
+      return files.indexOf(file) == index;
+    });
```

Comment:
`standard` will fail on this, otherwise LGTM!

Reply:
@ralphtheninja Thanks! Embarrassing to do an indentation miss. :)

Fixed the linting issues and amended to the commit.